src/app/components/videochat/jh-video-chat.html

```diff
@@ -19,7 +19,7 @@
   </header>
 </div>
 
-<div class="row">
+<div class="row" id="body">
```

Comment:
hmm, there is header above and footer below, so for me body is native name from old school pure html times :)